audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/AudioRecorderViewModel.kt

```diff
@@ -1,29 +1,46 @@
 package org.odk.collect.audiorecorder.recording
 
+import android.app.Application
+import android.content.Intent
+import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
-import org.odk.collect.audiorecorder.recorder.Recorder
+import org.odk.collect.audiorecorder.recording.AudioRecorderService.Companion.ACTION_CANCEL
+import org.odk.collect.audiorecorder.recording.AudioRecorderService.Companion.ACTION_START
+import org.odk.collect.audiorecorder.recording.AudioRecorderService.Companion.ACTION_STOP
 import java.io.File
 import javax.inject.Inject
 
-internal class AudioRecorderViewModel(private val recorder: Recorder) : ViewModel() {
+internal class AudioRecorderViewModel(private val application: Application, private val recordingRepository: RecordingRepository) : ViewModel() {
+
+    val recording: LiveData<File?> = recordingRepository.getRecording()
 
     fun start() {
-        recorder.start()
+        application.startService(
```

Comment:
I'm not 100% sure about the ViewModel being responsible for the service like this but it feels good to hide that there is a Service involved from the UI.